libs/sof/sof_multicamera_gpu.cpp

```
@@ -66,60 +68,112 @@ void MultiSOFGPU::LaunchTrackingPrimaryToSecondary(CameraId primary_id, CameraId
+  }
 
-    TrackData& data = tracks_data[i];
-    Vector2T uvL;
-    intrinsicsP.denormalizePoint(xyL, uvL);
+  size_t tracked_count = 0;
+
+  // Multi-launch scan: iterate candidate index k = 0..max_candidates-1. Points that have already
+  // succeeded, or that ran out of candidates, are given a benign (zero) offset — the kernel still
+  // processes them (batched-launch limitation) but their results are discarded host-side. First
+  // successful candidate wins, matching CPU semantics.
+  for (size_t k = 0; k < max_candidates; ++k) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)